### PR TITLE
add permissions field to proto

### PR DESCRIPTION
**Pull request by Kschappacher**
add permissions field to invite requests

### CODE REVIEW

**Comment by edaniels** [comment]
what is this?

File: proto/viam/app/v1/app.proto [hunk 1]
```diff
@@ -254,6 +254,7 @@ message ListOrganizationMembersResponse {
 message CreateOrganizationInviteRequest {
   string organization_id = 1;
   string email = 2;
+  string permissions = 3;
```
[the remark this answers is not in the document]
Comment (michaellee1019) [reply proto/viam/app/v1/app.proto:257]: Also if you could remove Authorization.identity_id in this PR that would be great. One change from tech spec.

[the remark this answers is not in the document]
Comment (Kschappacher) [reply proto/viam/app/v1/app.proto]: Do you only want identity_id removed from just Authorization? or do you want it removed from AddRoleRequest and RemoveRoleRequest as well?

**Comment by michaellee1019** [comment]
@Kschappacher lets bring in latest changes from main. After that should just be a single line change to add:
`repeated Authorization authorizations`

**Pull request closed without merging**